Traveler/ViewControllers/ProfileViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  ProfileViewController.swift
+//  Traveler
+//
+//  Created by Dorothy Fu on 2019-03-19.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+import TravelerKitUI
+
+
+class ProfileViewController: UITableViewController {
+    @IBOutlet weak var image: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+
+    var profile: Profile?
+
+    override func viewDidLoad() {
+        guard profile != nil && profile?.firstName != nil else {
+            return
+        }
+
+        nameLabel.text = "\(profile?.firstName ?? "")\(profile?.lastName ?? "")"
+    }
+
+    open override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        switch (segue.identifier, segue.destination) {
+        case (_, let vc as SettingsViewController):
+            vc.profile = profile
+        default:
+            Log("Unknown segue", data: nil, level: .warning)
+            break
+        }
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
+        case 0:
+            guard profile?.firstName != nil else {
+                return 1
+            }
+            return 2
+        case 1:
+            return 2
+        default:
+            Log("Invalid Section", data: section, level: .error)
+            return 0
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        switch indexPath.section {
+        case 0:
+            return 60
+        default:
+            return 44
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch (indexPath.section, indexPath.row) {
+        case (0, _):
+            let cell = tableView.dequeueReusableCell(withIdentifier: "profileCellIdentifier", for: indexPath) as! ProfileCell
+            if profile?.firstName != nil && indexPath.row == 0 {
+                cell.titleLabel.text = "Name"
+                cell.valueLabel.text = "\(profile?.firstName ?? "")\(profile?.lastName ?? "")"
```

Comment:
No space between fName and lastName?
DorothyFu

Reply:
It comes with a trailing space on the first name @Skrinikov

Reply:
Well thats intuitive

Reply:
@GeorgiosP will trim extra white space and iOS + Android will add a space